src/devices/tfa_30.3211.c

```diff
@@ -0,0 +1,101 @@
+#include "rtl_433.h"
+#include "util.h"
+#include "data.h"
+
+/*
+ TFA 30.3211.02 
+ 
+ 1970us pulse with variable gap (third pulse 3920 us)
+ 
+ Above 79% humidity, gap after third pulse is 5848 us
+ 
+ Bit 1 : 1970us pulse with 3888 us gap
+ Bit 0 : 1970us pulse with 1936 us gap
+ 
+ Demoding with -X "tfa_test:OOK_PPM_RAW:2900:5000:36500"
+ 
+ 74 bit (2 bit preamble and 72 bit data => 9 bytes => 18 nibbles)
+ 
+ Nibble       1   2    3   4    5   6    7   8    9   10   11  12   13  14   15  16   17  18
+           PP ?HHHhhhh ??CCNIII IIIITTTT ttttuuuu ???????? ???????? ???????? ???????? ??????
+ 
+     P = Preamble
+     H = First digit humidity 7-bit 0=8,1=9 (Range from 20 - 99%)
+     h = Second digit humidity
+     C = Channel
+     T = First digit temperatur
+     t = Second digit temperatur
+     u = Third digit temperatur
+     N = Negative temperatur
+     
+
+ */
+
+static int tfa_303211_callback (bitbuffer_t *bitbuffer) {
+    data_t *data;
+    uint8_t *b      = bitbuffer->bb[0];
+    const uint16_t length = bitbuffer->bits_per_row[0];
+
+    /* length check */
+	if (74 != length) {
+		if(debug_output) fprintf(stderr,"tfa_303211 wrong size (%i bits)\n",length);
+		return 0;
+	}
+
+    /* shift preamble */
+    int i;
+    for (i = 0; i < 9; i++) {
+      uint8_t b1 = b[i] << 2;
+      uint8_t b2 = (b[i+1] & 0xC0) >> 6;
+      b[i] = b1 | b2;
+    }
+
+    const float temp        = ((b[2] & 0x0F) * 10) + ((b[3] & 0xF0) >> 4) + ((b[3] & 0x0F) *0.1F);
+    const int minus         = (b[1] & 0x08) >> 3;
+    const int hum           = (b[0] & 0x70) >> 4;
+    const int humidity      = (hum < 2 ? hum + 8 : hum) * 10 + (b[0] & 0x0F);
+    const int sensor_id     = ((b[1] & 0x07) << 4) | ((b[2] & 0xF0) >> 4);
+    const int battery_low   = 0;
+    const int channel       = (b[1] & 0x30) >> 4;
+
+    float tempC = (minus == 1 ? temp * -1 : temp);
+    {
+        char time_str[LOCAL_TIME_BUFLEN];
+        local_time_str(0, time_str);
+
+        data = data_make("time",          "",            DATA_STRING, time_str,
+            "model",         "",            DATA_STRING, "TFA 30.3211.02",
```

Comment:
place the first args on a new line (nicer alignment), use two indents (8 spaces)